packages/react-dom-bindings/src/client/ReactDOMFloatClient.js

```diff
@@ -795,8 +969,104 @@ function createPreloadResource(
   };
 }
 
+function acquireHeadResource(resource: HeadResource): Instance {
+  resource.count++;
+  let instance = resource.instance;
+  if (!instance) {
+    const {props, root, type} = resource;
+    switch (type) {
+      case 'title': {
+        const titles = root.querySelectorAll('title');
```

Comment:
You can use `getElementsByTagName` in this case.

Reply:
Probably leftover for when I was removing the title nodes and needed a static list. I’ll change